source/_integrations/nws.markdown

```diff
@@ -25,6 +25,8 @@ According to the [API documentation](https://www.weather.gov/documentation/servi
 
 Providing a METAR station code is optional, and if not supplied, the closest station to the latitude and longitude will be chosen. A list of nearby stations is printed to the log with level `DEBUG` if no station is supplied. Stations can also be found on the [NOAA website](https://www.cnrfc.noaa.gov/metar.php). Codes with only three characters, for example, `ADW` should be prefixed with the letter K, `KADW`.
 
-Two weather entities are created for each entry in the configuration: one for day and night forecasts and one for hourly forecasts. The hourly forecast entity is disabled after configuration but can be enabled by the user. The time supplied for each forecast is the start time for the forecast. Sensors are also created as disabled entities after configuration and can be enabled by the user.
+One weather entity is created to provide forecasts for each entry in the configuration. The time supplied for each forecast is the start time for the forecast. Hourly forecast data can be obtained using the `weather.get_forecasts` service. Sensors are also created as disabled entities after configuration and can be enabled by the user.
```

Comment:
```suggestion
One weather entity is created for each entry in the configuration with hourly and day/night forecasts, which can be obtained through the `weather.get_forecasts` service. The time supplied for each forecast is the start time for the forecast. Sensors are also created as disabled entities after configuration and can be enabled by the user.
```

Reply:
I can commit that if you especially like it, but "with" threw me off briefly, so it might make sense to replace the word "with" using the word "providing".   I'm not sure that comma is grammatically correct, either, so maybe it should be broken into more sentences like this?:
> One weather entity is created for each entry in the configuration.  This entity provides hourly and day/night forecasts which can be viewed in the UI or accessed using the `weather.get_forecasts` service.

Even if that's better, I'm not sure if I should say "the UI" or "lovelace" though, so let me know what you think.